docs/guides/operator/installation.adoc

```diff
@@ -62,7 +64,21 @@ kubectl apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resourc
 kubectl apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/{version}/kubernetes/kubernetes.yml
 ----
 
-Currently the Operator watches only the namespace where the Operator is installed.
+The Operator will watch the namespace where it is installed. You may optionally select a namespace with the `-n` option. 
 </@profile.ifCommunity>
 
+=== Installing Multiple Operators
+
+It is currently not fully supported for the operator to watch multiple or all namespaces. In circumstances where you want to watch multiple namespaces, you can install multiple operators.
+
+If you do this please be aware:
+
+- all Operators share the CRDs (Custom Resource Definitions) as they are installed cluster wide.
+- CRD revisions from newer Operator versions won't introduce breaking changes except for the eventual removal of fields that have been well deprecated. Thus newer CRDs are generally backward compatible.
+- the CRDs installed last will be the ones in use.
+- older CRDs may not be forwards compatible with new fields used by newer operators. When using OLM it will check if your custom resources are compatible with the CRDs being installed, so the usage of new fields can prevent the simultaneous installation of older operator versions.
+- fields introduced by newer CRDs will not be supported by older Operators. Older Operator will fail to handle such CRs with an error deserializing an unrecognized field.
```

Comment:
Our CRs and CRDs currently do not make use of additionalProperties (like fabric8 built-in objects) - if they ever do, then this last point won't hold.

Reply:
I believe failing is a more appropriate handling of unrecognized fields in the Operator as it highlights a configuration error (user tries to use a configuration incompatible with used Operator version).